src/wallet/rpcwallet.cpp

```diff
@@ -719,9 +719,18 @@ static UniValue getbalance(const JSONRPCRequest& request)
     auto locked_chain = pwallet->chain().lock();
     LOCK(pwallet->cs_wallet);
 
-    const UniValue& dummy_value = request.params[0];
-    if (!dummy_value.isNull() && dummy_value.get_str() != "*") {
-        throw JSONRPCError(RPC_METHOD_DEPRECATED, "dummy first argument must be excluded or set to \"*\".");
+    const UniValue& trusted_only_uv = request.params[0];
+    bool trusted_only = true;
+    if (trusted_only_uv.isBool()) {
+        trusted_only = trusted_only_uv.get_bool();
+    } else if (trusted_only_uv.isStr()) {
```

Comment:
This won't work now because of the entry in vRPCConvertParams. I don't think a single parameter can accept either a string or a boolean in this way, because there is no way to tell otherwise if `true` is supposed to be a string `"true"` (which should error) or the boolean value `true` (which should work). I think the only options are either to break compatibility and drop the `"*"` case in favor of the boolean only, or not switch to a boolean and just accept `"*"` and `""` (empty string)

Reply:
It works fine... the lookup table has no effect on the JSON-RPC server, only the bitcoin-cli testing tool.